common/courses/participants.ts

```diff
@@ -73,10 +73,16 @@ export async function joinSubcourseWaitinglist(subcourse: Subcourse, pupil: Pupi
     if (await isParticipant(subcourse, pupil)) {
         throw new RedundantError(`Pupil is already participant`);
     }
+    if (await isOnWaitingList(subcourse, pupil)) {
+        throw new RedundantError('Pupil is already on waiting list');
+    }
 
     try {
-        await prisma.subcourse_waiting_list_pupil.create({
-            data: { pupilId: pupil.id, subcourseId: subcourse.id },
+        await prisma.waiting_list_enrollment.create({
+            data: {
+                subcourseId: subcourse.id,
```

Comment:
Does Prisma set createdAt?

Reply:
Yes, it should: https://github.com/corona-school/backend/blob/44cabf280ed7b8c0d9dfff7611e05dbfbbbd5b77/prisma/schema.prisma#L659